### ISSUE TITLE
Embedding metadata is not read or displayed in TensorBoard 2.3.0

### ISSUE THREAD

**Issue by toddwasson** [tensorflow/tensorflow#44057]
Hello, I have trained a model (with TensorFlow 2.3.1) including embeddings via `keras.layers.Embedding` and prepared label files per embedding dictionary, which I point to via the `embeddings_metadata` parameter of `keras.callbacks.TensorBoard`.  TensorBoard executes and displays the various embedding vectors in the Projector without issue, but the embeddings are not labeled per the label files.  The label files are successfully created and stored in the log directory, and I can use them manually just fine via the `Load` button in the TensorBoard interface.

I debugged the `projector_plugin.py` code and determined that `_get_metadata_file_for_tensor()` is never called (verified via `prints`, debugging with `pdb`, etc.).  This method looks to be the only place in the code used to retrieve the `metadata_path` from which the labels would be read, and is apparently only accessed via the `/metadata` HTTP route.

Incidentally when I add the various `Embedding` layers in Keras, the first receives the `tensor_name` `layer_with_weights-0/embeddings/.ATTRIBUTES/VARIABLE_VALUE` but the subsequent ones are `layer_with_weights-1`, `2`, etc.  I see the `keras.callbacks.TensorBoard` hard-codes them to `layer_with_weights-0` but I think this is a separate issue as I have manually corrected the `projector_config.pbtxt` file and, as I mentioned above, the `_get_metadata_file_for_tensor()` method is never called anyway so they aren't even trying to be parsed and used to label tensors.

Any thoughts?  Am I missing something?

Thanks!

Todd

**Comment by ravikyram** [comment]
@toddwasson 

This issue is more suitable for Tensorboard repo.Please post it on [Tensorboard repo](https://github.com/tensorflow/tensorboard/issues/new/choose) from here. Thanks!

**Comment by toddwasson** [comment]
Oops, that's what I was trying to accomplish -- thanks for pointing me in the right direction!

**Comment by toddwasson** [comment]
In case anyone comes looking in the future, I did create [a TensorBoard repo issue](https://github.com/tensorflow/tensorboard/issues/4246).

**Comment by ravikyram** [comment]
@toddwasson 

Can we close the issue here and track the issue in Tensorboard repo issue you have created. Thanks!

**Comment by toddwasson** [comment]
Yes, let's close it.  Thanks again for your help!
